tests/lmeval/test_lmeval.py

```diff
@@ -127,7 +127,12 @@ def test_lm_eval(self, test_data_file: str):
         base_results = self._eval_base_model()
 
         if not self.save_dir:
-            self.save_dir = self.model.split("/")[1] + f"-{self.scheme}"
+            base_name = self.model.split("/")[1] + f"-{self.scheme}"
+            hf_cache = os.environ.get("HF_HUB_CACHE")
```

Comment:
```python3
from huggingface_hub import constants
constants.HF_HUB_CACHE
```

FYI this is what transformers uses, and I think this should be more robust, as it [handles expanding the path and fallbacks](https://github.com/huggingface/huggingface_hub/blob/7e8595b6b1728e392bda85c9d151613b50112434/src/huggingface_hub/constants.py#L146)